cmd/horcrux/cmd/migrate_test.go

```diff
@@ -0,0 +1,147 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMigrateV2toV3(t *testing.T) {
+	v2ConfigBz, err := os.ReadFile("./fixtures/config-v2.yaml")
```

Comment:
Shouldn't happen in this PR, but typically this would be reading from a `testdata` directory, as that name has special meaning to the Go toolchain.

Furthermore, unless this actually _needs_ to interact with disk, I prefer to lean on `//go:embed` here, so that if we ever need a test binary, it isn't dependent on the local filesystem. (It works with `go test` because that commands sets the working directory to the package being tested.)

Reply:
+1 for `testdata` dir and `go:embed`

Reply:
definitely nice cleanup. migrated